index.html

```diff
@@ -3078,25 +3078,62 @@ <h3>Extensibility</h3>
         <section>
           <h4>Semantic Interoperability</h4>
 
+          <p>
+            When defining new terms in an application-specific vocabulary, developers MUST use globally unambiguous [=URLs=] to identify the terms.
+            Whenever possible, it is RECOMMENDED to re-use terms — and their corresponding URLs — defined by well-known, public vocabularies,
+            such as [[[?DCTERMS]]] [[?DCTERMS]] or [[[?schema-org]]] [[?schema-org]]. 
+            If that is not possible, developers MUST <em>define</em> a new URL for each term. 
+            When doing so, general [[LINKED-DATA]] principles must be followed, in particular:
+          </p>
+
           <ul>
-            <li>
-JSON-LD-based processors MUST produce an error when a JSON-LD context redefines
-any term in the
-<a href="https://www.w3.org/TR/json-ld/#dfn-active-context">active context</a>.
-The only way to change the definition of existing terms is to introduce a new
-term that clears the active context within the scope of that new term. Authors
-that are interested in this feature should read about the
-`@protected` feature in the JSON-LD 1.1 specification.
-            </li>
-          </ul>
+              <li>
+                Human-readable documentation MUST be published, describing the semantics of and
+                the constraints on the use of each term.
```

Comment:
This section is turning into a "Requirements for Vocabulary Document Authors", which I thought we were going to avoid writing as of last week. IOW, this PR feels like it's trying to also address issue https://github.com/w3c/vc-data-model/issues/1410

That said, this isn't a bad list (although, feels a bit heavy handed)...